src/logic/safe/store/actions/processTransaction.ts

```diff
@@ -91,9 +82,10 @@ export const processTransaction = (props: ProcessTransactionArgs): ProcessTransa
       data: txProps.txData,
       gasPrice: tx.gasPrice || '0',
       sender: sender.from,
-      sigs:
-        generateSignaturesFromTxConfirmations(tx.confirmations, preApprovingOwner) ||
-        getPreValidatedSignatures(sender.from),
+      sigs: generateSignaturesFromTxConfirmations(
```

Comment:
Why do we no longer need prevalidated signatures? Because it is processing?

Reply:
Because it's actually called as part of `generateSignaturesFromTxConfirmations`. If confirmations are 0, it will just use the owner address as a sig. So it was just redundant to have this conditional here.